solr/Dockerfile.okd

```diff
@@ -0,0 +1,59 @@
+FROM solr:7.7.2-slim
+
+#COPY --chown=solr:solr config /opt/solr/server/solr/persons
+#COPY --chown=solr:solr config /opt/solr/server/solr/organizations
+#COPY --chown=solr:solr config /opt/solr/server/solr/documents
+#COPY --chown=solr:solr config /opt/solr/server/solr/relationships
+#COPY --chown=solr:solr config /opt/solr/server/solr/processes
+#COPY --chown=solr:solr config /opt/solr/server/solr/collections
+#COPY --chown=solr:solr config /opt/solr/server/solr/concepts
+
+RUN chmod ugo+w -R /opt/solr/server/solr/
+RUN chmod ugo+w -R /opt/solr/server/logs/
+
+USER solr
+
+RUN mkdir -p /opt/solr/server/solr/persons
```

Comment:
hi @ht29 maybe we should add a commented section here explaining a little why we had to separate these calls. something like:

```
# calls had to be separated into multiple commands
# in order to support OpenShift environment.
```

Reply:
Hi @outten45 , I think it is a good idea.. I can add some meaningful comments in there for explaination...thanks